data/events.txt

```diff
@@ -3333,6 +3334,7 @@ event "syndicate tech available"
 
 event "geminus rebuilt"
 	planet Geminus
+		attributes paradise shipyard syMajor factory mining
```

Comment:
the shipyard attribute did not exist as attribute until today either. I created all three (shipyard, syMinor, syMajor) and also added a documentation header to the map.txt file summarizing all currently used attributes.
syMajor describes "home of (xyz) shipyard", like home of the Navy shipyard.